integrations/extensions/starter-kits/coveo/README.md

```diff
@@ -39,73 +41,74 @@ If you want to add this starter kit to an _existing_ assistant, you cannot use t
-- Go to `Variables > Created by you` and add `query_text`, `result_items` `link0`, `link1`, `link2`, `title0`, `title1`, `title2`, `snippet0`,`snippet1`, and `snippet2`.
+- Go to `Variables > Created by you` and add `query_text`, `search_results`, `search_result`, `link`, `title`, and `snippet`.
 - Create a new action and put "Search" in "What does your customer say to start this interaction?".  Add step 1:
-    - Click the fX button to add a variable and add new session variable `query_text` and select "Expression" type and then put `input.text` or `input.original_text` as the expression.  The former will employ spelling correction to fix any detected spelling errors before sending the query, which can be helpful, but it can also be counterproductive if your documents include specialized terminology that is not in our dictionary (such as product names) so you can use `input.original_text` as the alternative in such cases.
-    - Optional: In "Assistant says", put `Searching for: ${query_text}`
-    - In "And then", select "Use an extension", select the extension you made back in step 2, and select the search endpoint and set the `q` parameter to the `query_text` session variable.
+    - Click the fX button to add a variable and add new session variable `query_text` and select "Expression" type and then put `input.original_text` as the expression.
+    - Add step 2, and change "without conditions" to "with conditions" and set the condition `query_text` is not defined. Then set the variable `query_text` to `input.text`. As noted in the [documentation for spell checking](https://cloud.ibm.com/docs/assistant?topic=assistant-dialog-runtime-spell-check#dialog-runtime-spell-check-how-it-works), `input.original_text` is set _only_ if the utterance from the user was altered due to spell correction and then it records the original request from the user and not the spell corrected text.  Spell correction can be very counter productive for searching because it can take specialized domain vocabulary and "correct" those terms to generic words in the language, so it is often better to apply the search on the original text.  The next step in Search then checks to see if the `query_text` is set (which it will be only if spell correction activated) and if not, it falls back to `input.text`.  At this point, `query_text` is guaranteed to be the exact original query issued by the user. The next calls the search extension. Using `input.text` for the query will employ spelling correction to fix any detected spelling errors before sending the query. This can be counterproductive if your documents include specialized terminology that is not in our dictionary (such as product names), which is why it's set up in the starter kit as a backup, in case you prefer not to use `input.original_text`.
```

Comment:
The text of the first two bullets winds up being a little confusing I think because it mixes together the instructive text from this kit with my text from the WD README (which is just describing and not providing instructions).  Here is a proposed rewrite:

- Click the fX button to add a variable and add new session variable `query_text` and select "Expression" type and then put `input.original_text` as the expression.  As noted in the [documentation for spell checking](https://cloud.ibm.com/docs/assistant?topic=assistant-dialog-runtime-spell-check#dialog-runtime-spell-check-how-it-works), `input.original_text` is set if the utterance from the user was altered due to spell correction and then it records the original request from the user and not the spell corrected text.  Spell correction can be very counter productive for searching because it can take specialized domain vocabulary and "correct" those terms to generic words in the language, so it is often better to apply the search on the original text, as we are doing here.
- Add step 2, and change "without conditions" to "with conditions" and set the condition `query_text` is not defined. Then set the variable `query_text` to `input.text`.   This is needed because the `input.original_text` is _only_ set when spell correction changed the text.  When there was no spell correction, you need to use `input.text` instead.  After step 2, `query_text` is guaranteed to be the exact original query issued by the user.